be/src/olap/schema.h

```diff
@@ -143,24 +154,30 @@ class Schema {
         *reinterpret_cast<bool*>(row + _col_offsets[index]) = true;
     }
 
-    void set_not_null(int index, char*row) const {
+    void set_not_null(int index, char* row) const {
         *reinterpret_cast<bool*>(row + _col_offsets[index]) = false;
     }
 
+    void set_is_null(void* row, uint32_t cid, bool is_null) const {
```

Comment:
The parameter is not match with set_not_null

Reply:
done